codegen/testserver/generated.go

```diff
@@ -938,7 +939,11 @@ func (ec *executionContext) _Circle_radius(ctx context.Context, field graphql.Co
 	}
 	res := resTmp.(float64)
 	rctx.Result = res
-	return graphql.MarshalFloat(res)
+	resMarshaler := func() graphql.Marshaler {
+		return graphql.MarshalFloat(res)
+	}()
+	ec.Tracer.EndFieldExecution(ctx)
```

Comment:
rather than needing to wrap all the marshallers in another func, could we `defer ec.Tracer.EndFieldExecution(ctx)`?